### ISSUE TITLE
# DAAN Why each epoch needs to redefine the optimizer in train.py line 77

### ISSUE THREAD

**Issue by Vpredictor** [jindongwang/transferlearning#366]
![image](https://user-images.githubusercontent.com/30548695/208914510-e79bdbc4-efe9-4d3e-9d6a-8ab8067a3b38.png)
I mean why not define in the main loop?

**Comment by jindongwang** [comment]
Because in Line 75 we see that learning rate is updated by epoch.
